src/componentfuncs/Comments.jsx

```diff
@@ -19,4 +23,56 @@ export function RetrieveComments (props) {
   }
 }
   
+export function LeaveComment (props) {
+  const [comment, setComment] = useState()
+  const [validity, setValid] = useState(true)
+  const [inputFeedback, setInputFeedback] = useState('success')
+
+
+  const showing = props.show
+  const currentId = props.id
+
```

Comment:
This is a strange variable name (commentValidate)... Is this function's only job to validate a comment?